FrontEnd/index.html

```diff
@@ -149,26 +149,26 @@ <h2>Galerie photo</h2>
           </button>
         </div>
         <h2>Ajout photo</h2>
-        <div id="workAddition">
-          <img class="pictureUpload" src="assets/images/Group(1).svg" alt="Ajouter une photo">
-          <label>+ Ajouter photo
-            <input id="modalAdditionInput"
-            type="file"
-            class="modalAdditionButton"
-            id="modalAdditionBtn"
-            />
-          </label>
-          <p>jpg, png : 4mo max</p>
-        </div>
         <form id="addWorkForm" action="#" method="post">
-          <label for="title">Titre</label>
-          <input type="text" name="title" id="title" />
-          <label for="categorySelection">Catégorie</label>
-          <select name="categorySelect" id="selectCategory">
-           <!--<option value="1">Objets</option>
-            <option value="2">Appartements</option>
-            <option value="3">Hotels & restaurants</option>-->
-          </select>
+          <div id="workAddition">
+            <article id="uploadedImgBox">
+              <img class="pictureUpload" src="assets/images/Group(1).svg" alt="Ajouter une photo">
```

Comment:
If the `(1)` is not mandatory for the semantic, you could change the file name ^^

```suggestion
              <img class="pictureUpload" src="assets/images/Group.svg" alt="Ajouter une photo">
```